src/main/java/org/broadinstitute/hellbender/tools/AnnotatePairOrientation.java

```diff
@@ -33,11 +33,11 @@
 
 @DocumentedFeature
 @CommandLineProgramProperties(
-        summary = "(Experimental) This adds fields normally emitted by M2 to a VCF.  There should never be a need to run this tool on a VCF that was produced by M2." +
+        summary = "This adds fields normally emitted by M2 to a VCF.  There should never be a need to run this tool on a VCF that was produced by M2." +
                 "\n  The output of this tool should be usable with FilterByOrientationBias." +
                 "\n  The output of this tool only counts reads that fully overlap (and match) the variant or reference sequence (this is relevant for indels)." +
                 "\n  IMPORTANT:  This tool does not produce the exact same F1R2/F2R1 as M2, due to the nature of how M2 calls variants (using read likelihoods, whereas this tool uses a base quality filter).",
-        oneLineSummary = "(EXPERIMENTAL) Annotate a non-M2 VCF (using the associated tumor bam) with pair orientation fields (e.g. " + GATKVCFConstants.F1R2_KEY + " ).",
```

Comment:
Should this tool be marked as `@ExperimentalFeature`? (it's currently marked as `@BetaFeature`)

Reply:
@LeeTL1220 @sooheelee AnnotatePairOrientation is currently annotated as `@BetaFeature`, but contains the word "Experimental" in summary text (removed in this PR). But the [comments in the test code](https://github.com/broadinstitute/gatk/blob/67f0f0f2e59185b721398b17c24eba487a2ac76c/src/test/java/org/broadinstitute/hellbender/tools/AnnotatePairOrientationIntegrationTest.java#L33) indicate it should still be marked `EXPERIMENTAL`. Whcih should it be ?

Reply:
@cmnbroad, I'm curious why is this functionality is a standalone tool and not an annotation offered by VariantAnnotator? Also, we should be sure to differentiate this annotation from others that are similar in the javadoc, e.g. [OxoGReadCounts](https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_walkers_annotator_OxoGReadCounts.php), if indeed they are different. If they are the same, then we need to deduplicate. 

Also, fyi, we already have in place a note on caveats when annotating externally produced VCFs at https://software.broadinstitute.org/gatk/documentation/article?id=10836, which we direct folks to when they expand the _Variant Annotations_ section of the [tool documentation page](https://software.broadinstitute.org/gatk/documentation/tooldocs/current/). 

As for whether this tool should be marked EXPERIMENTAL, I leave that to @LeeTL1220 to decide. One thing perhaps to followup on is whether the putative bug reported in [this forum thread](https://gatkforums.broadinstitute.org/gatk/discussion/23246/the-annotatepairorientation-tool-and-non-mutect2-vcf-input-only-0-0-0-0-values-in-the-output) has been solved. Adelaide informs me @kachulis could perhaps shed light on this, as he was the gatk-userliason at the time of this post.

Reply:
@sooheelee I never got anywhere investigating that issue, as I was waiting on the user to send some example data that exhibited the reported behavior.  Looking at the forum post, it looks like the user did eventually send some data along some weeks later, but I don't know if there was any investigation by someone else since then.  I would be happy to take a look if this hasn't already been figured out.

Reply:
Yes, the researcher sent Adelaide data and she just sent it along to me this evening when I followed up on the status. I can send the data along to you if you will follow up.

Reply:
Sure, I can take a look tomorrow.

Reply:
Great. I slacked you the data.

Reply:
Took a look today, and the issue was in the user's data, not a bug in the tool.